201-timeseriesinsights-environment-with-eventhub/azuredeploy.json

```diff
@@ -194,16 +208,50 @@
           },
           "tags": "[variables('eventSourceTagsValue')]",
           "dependsOn": [
-            "[concat('Microsoft.TimeSeriesInsights/environments/',parameters('environmentName'))]",
+            "[concat('Microsoft.TimeSeriesInsights/environments/', parameters('environmentName'))]",
             "[concat('Microsoft.EventHub/namespaces/', parameters('eventHubNamespaceName'), '/eventHubs/', parameters('eventHubName'), '/consumerGroups/', parameters('consumerGroupname'))]"
           ]
         }
       ]
+    },
+    {
+      "condition": "[if(equals(first(parameters('accessPolicyReaderObjectIds')), 'thisArrayIsReallyEmpty'), bool('false'), bool('true'))]",
```

Comment:
simplify:

"condition": "[not(empty(parameters('accessPolicyReaderObjectIds')))]"

Reply:
I can't simplify this one because of the problem with copy not excepting a length of 1.